### PR TITLE
3D Surface Doc

### PR DESCRIPTION
**Pull request by priyatharsan**
related to #423

### CODE REVIEW

File: _posts/3d/2016-08-18-how-to-make-a-3d-surface-plot.md [hunk 1]
```diff
@@ -1,64 +1,63 @@
...
-      img: "![Type directly axis](../static/images/3d-surface-plot-with-excel/final-plot.png)"
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
+    - copy: "Use the 'General' section under the 'Style' menu to set the plot title, as well as change the layout background, margin color and font styles."
+    - copy: "To set the plot title, type the title text within the textbox provided under the 'Title' property."
+      img: "![Type title](../static/images/3D-surface-plot/3D-surface-title.png)"
+    - copy: "Another approach is to click and then enter the title directly on the plot interface."
+      img: "![Type directly title](../static/images/3D-surface-plot/3D-surface-title-direct.png)"
+    - copy: "Use the 'Traces' section under the 'Style' menu to change the properties of the 3D surface such as the colorscale, lighting effects and its position, and hoverinfo."
```
Comment (ashmpace) [inline _posts/3d/2016-08-18-how-to-make-a-3d-surface-plot.md]: Position of the lighting effects, or position of the 3D surface?
Suggestion: "lighting effects, position and hoverinfo."

Comment (priyatharsan) [reply _posts/3d/2016-08-18-how-to-make-a-3d-surface-plot.md]: @ashmpace actually it refers to the "light position". Shall I replace with "lighting effects, light position and hoverinfo." ?

Comment (ashmpace) [reply _posts/3d/2016-08-18-how-to-make-a-3d-surface-plot.md]: @priyatharsan Yes, that's better. Same for other instances of that phrase as well.

Comment (priyatharsan) [reply _posts/3d/2016-08-18-how-to-make-a-3d-surface-plot.md]: 👍

**Comment by priyatharsan** [comment]
@ashmpace made the changes. Thanks.

**Pull request merged**